### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by ddjessicay**
Updating docs to reflect feature release

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/infrastructure/resource_catalog/_index.md [hunk 1]
```diff
@@ -61,9 +61,9 @@ The security tab allows you to gain a clear understanding of resources with secu
...
+You can sort resources in the Resource Catalog list by cloud platform, resource type, account, team, region, misconfigurations, and threats. Sort by **Threats** to spot workloads impacted over the past four hours. Sort by **Misconfigurations** to identify cloud resources most prone to misconfigurations. In addition, you can export your Resource Catalog list as a CSV file from the top right corner of the list.
 
-To find a particular resource, search by its name. To filter the list to see a subset of resources you're most interested in, select facets on the left panel. For example, you may find it helpful to filter by your team name, or to scope the misconfigurations to particular environments and clusters.
+To find a particular resource, search by its name. To filter the list to see a subset of resources you're most interested in, select facets on the left panel. For example, you may find it helpful to filter by your team name, or to scope the misconfigurations to particular environments and clusters. To quickly access the relevant cloud console, select the cloud console icon for the desired resource.
```
Comment (buraizu) [inline content/en/infrastructure/resource_catalog/_index.md:66]:
```suggestion
To find a particular resource, search by its name. To filter the list to see a subset of resources you're most interested in, select facets on the left panel. For example, you may find it helpful to filter by your team name, or to scope the misconfigurations to particular environments and clusters. To access the relevant cloud console, select the cloud console icon for the desired resource. 
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for the PR; just requesting a minor update to comply with our style guide

**Review by buraizu — APPROVED** [review]

**Comment by buraizu** [comment]
@ddjessicay thanks for making that update. Let me know if this change is ready to be merged

**Comment by ddjessicay** [comment]
@buraizu Yes it's ready to be merged, sorry I may have clicked merge master branch into this branch - hopefully this doesn't result in any errors! Thanks

**Pull request merged**